pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6_test.go

```diff
@@ -0,0 +1,103 @@
+package dhcpv6
+
+import (
+	"github.com/insomniacslk/dhcp/dhcpv6"
+	"github.com/insomniacslk/dhcp/iana"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"net"
```

Comment:
The order of the imports is not correct, afaiu.

Reply:
why? I never order the imports manually, AFAIU, it is done by the `make`.

Reply:
The first block has the golang core imports, the second block has github imports, the 3rd block has kubevirt imports.

But, as you correctly pointed out, since the linter ate it, I can't complain.

Reply:
done.